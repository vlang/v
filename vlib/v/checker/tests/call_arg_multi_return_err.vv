fn some_fn() (int, int) {
	return 0, 0
}

fn some_fn2() {
	ret := some_fn()
	some_fn3(ret)
}

fn some_fn3(param int) {}
