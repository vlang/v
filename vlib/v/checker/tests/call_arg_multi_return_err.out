vlib/v/checker/tests/call_arg_multi_return_err.vv:6:6: error: assignment mismatch: 1 variable but `some_fn()` returns 2 values
    4 | 
    5 | fn some_fn2() {
    6 |     ret := some_fn()
      |         ~~
    7 |     some_fn3(ret)
    8 | }
vlib/v/checker/tests/call_arg_multi_return_err.vv:7:2: error: trying to pass 2 argument(s), but function expects 1 argument(s)
    5 | fn some_fn2() {
    6 |     ret := some_fn()
    7 |     some_fn3(ret)
      |     ~~~~~~~~~~~~~
    8 | }
    9 |
